assets/data-port/import/done/index.js

```diff
@@ -1,18 +1,24 @@
-import { __ } from '@wordpress/i18n';
-import { Section, H } from '@woocommerce/components';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { DonePage } from './done-page';
 
-/**
- * This component displays the final page when import has completed.
- */
-export const DonePage = () => {
-	return (
-		<section className="sensei-done-page">
-			<header className="sensei-data-port-step__header">
-				<H>{ __( 'Done', 'sensei-lms' ) }</H>
-			</header>
-			<Section component="section">
-				<p>Placeholder.</p>
-			</Section>
-		</section>
-	);
-};
+export default compose(
+	withSelect( ( select ) => {
+		const store = select( 'sensei/import' );
+
+		const { results, logs } = store.getStepData( 'done' );
+		return {
+			results,
+			logs,
+		};
+	} ),
+	withDispatch( ( dispatch, ownProps, { select } ) => {
+		const { fetchImportLog, restartImporter } = dispatch( 'sensei/import' );
+
+		return {
+			fetchImportLog: () =>
+				fetchImportLog( select( 'sensei/import' ).getJobId() ),
```

Comment:
Nice! I wonder if we could do something like this with `_.partial` for the other places we are passing down the job ID.

Reply:
Yeah that looks right. 375b739

Reply:
I don't think `_.partial` is needed unless there are other arguments, but doesn't harm anything either I suppose.

Reply:
One extra benefit here is the `jobId` is selected at render time, not when calling the function. Well, it feels like that's a benefit :)
Also easy to extend if something like paging parameters come in.